parameter/ParameterMgr.h

```diff
@@ -34,16 +34,15 @@
 #include <vector>
 #include "RemoteCommandHandlerTemplate.h"
 #include "PathNavigator.h"
-#include "SelectionCriterionType.h"
-#include "SelectionCriterion.h"
 #include "Element.h"
 #include "XmlDocSink.h"
 #include "XmlDocSource.h"
 #include "Results.h"
 #include "ParameterFrameworkConfiguration.h"
-#include "SelectionCriteria.h"
+#include "criterion/Criteria.h"
 #include "ConfigurableDomains.h"
 #include "SystemClass.h"
+#include <criterion/Criterion.h>
```

Comment:
This header still needed internally, maybe we can use a forward declaration as we are just returning a pointer on a Criterion object